### ISSUE TITLE
[Core Mechanic][Professions]Mining doesn't got a chance to skill up fresh tapping after the initial taping

### ISSUE THREAD

**Issue by Tenoutoften** [Atlantiss/NetherwingBugtracker#4045]
**Description**: 
Previously you could tap endlessly without looting and force skill up on every vein(https://github.com/Atlantiss/NetherwingBugtracker/issues/3890).
Now you can only get a skill up on the first tap. On the other taps you will never skill up(Despite looting and moving on to a fresh "tapping".)

**Current behaviour**:
You can only get a skill up on the first tap. On the other taps you will never skill up.

**Expected behaviour**: 
You should be able to get a chance to gain a skill on the other taps, provided they are "fresh".

example:

1) Mine a vein once but don't loot will give a chance to skill up - However mining it a second time without looting won't.
2) Mining a vein and looting without getting skill up - and then mining it a second time and looting should give a chance for skill up.

In other words:
If you haven't seen the loot within the Vein you will have a chance to skill up. If you have seen the loot, you will not have a chance to skill up.

**Server Revision**: 
2604

**Comment by wtRCH** [comment]
@Shanghi Any confirmation on this?

**Comment by staplerx300** [comment]
definitely not blizzlike, you should be able to get more skillups after the first one

**Comment by Tenoutoften** [comment]
someone else reported this later so ill close this one since it serves no purpose having two identical ones open...

**Comment by femtaur** [comment]
this issue forgotten? 26 days
